src/nvapi_vulkan.cpp

```diff
@@ -41,6 +41,10 @@ extern "C" {
             }
         }
 
+        if (!lowLatencyDevice->IsLayerPresent()) {
+            log::info("Initializing Vulkan Low-Latency failed: could not find VK_NV_low_latency2 commands in VkDevice's dispatch table, please ensure that DXVK-NVAPI's Vulkan layer is present. We will pretend that Reflex is working");
```

Comment:
Small suggestion to state this more explicit since I guess a lot of users will read this:
“… DXVK-NVAPI will pretend that Reflex is working, thus Reflex appears to be working for the application/game, but it isn’t and latency will be bad.”

Reply:
Minor bikeshedding: "... will still be bad" may be overly subjective.  It might actually be fine for many/most players, just not ideal.  IMHO the existing message is factual enough and actionable.